### PR TITLE
Okta ingestion

### PR DESCRIPTION
**Pull request by greg-snowflake**

### CODE REVIEW

File: src/setup.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(
     name="snowalert-runners",
     version='1.0',
-    packages=['runners', 'runners.helpers', 'runners.plugins'],
+    packages=find_packages(),
```
Comment (andrey-snowflake) [inline src/setup.py]: these should go away if you rebase

File: src/pipeline/okta_ingestion.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import alooma_pysdk
```
Comment (andrey-snowflake) [inline src/pipeline/okta_ingestion.py]: duplicate file?

File: src/ingestion/okta_ingestion.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                break
+            url = r.headers['Link'].split(', ')[1].split(';')[0][1:-1]
+        except Exception as e:
+            print(e)
+
+    alooma_pysdk.terminate()
+
+
+if __name__ == "__main__":
+    print("invoked!")
```
Comment (andrey-snowflake) [inline src/ingestion/okta_ingestion.py]: might make sense to add python-fire here and a --debug flag?

File: src/setup.py (hunk 1 above)
Comment (greg-snowflake) [reply src/setup.py]: Ran git fetch and git rebase.

File: src/pipeline/okta_ingestion.py (hunk 2 above)
Comment (greg-snowflake) [reply src/pipeline/okta_ingestion.py]: Ah, this was a casualty of the rename from pipeline to ingestion. Should be removed now.

File: src/ingestion/okta_ingestion.py (hunk 3 above)
Comment (greg-snowflake) [reply src/ingestion/okta_ingestion.py]: This was a holdover from when I was testing stuff; it doesn't need to make it into production code. Removed, thanks for catching.

File: src/setup.py (hunk 1 above)
Comment (andrey-snowflake) [reply src/setup.py]: `git fetch` updates your local db version of remote and `git rebase` rebases on the remote. he, I meant `git rebase v1.5.5`, which rebases this pull against the latest v1.5.5

File: src/ingestion/okta_ingestion.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+def process_logs(logs):
+    for l in logs:
+        ALOOMA_SDK.report(l)
+
+
+def get_timestamp():
+
+    # Once pipelines are more strongly integrated with the installer, this table should be a variable
+    timestamp_query = """
+        SELECT V:published from SECURITY.ALOOMA.OKTA
```
Comment (andrey-snowflake) [inline src/ingestion/okta_ingestion.py]: why not `{DATA_TABLE}.okta`?

**Pull request merged**